packages/@okta/vuepress-site/docs/guides/oie-upgrade-api-sdk-to-oie-sdk/main/index.md

```diff
@@ -167,25 +167,25 @@ curl --location --request POST 'https://${yourOktaDomain}/api/v1/sessions' \
 }
 ```
 
-If your application code implements these API calls and handles the responses shown, you need to update your code to use Identity Engine SDK methods. These methods encapsulate the authentication flow using recursive calls to the Identity Engine, and a successful response returns with access and ID tokens.
+If your application code implements these API calls and handles the responses shown, you need to update your code to use Identity Engine SDK methods. These methods encapsulate the authentication flow using recursive calls to Identity Engine, and a successful response returns with access and ID tokens.
 
-See [Okta Identity Engine SDK authentication flow](#okta-identity-engine-sdk-authentication-flow).
+See [Identity Engine SDK authentication flow](#identity-engine-sdk-authentication-flow).
```

Comment:
This doesn't work for asp.net and ios because there is no "Identity Engine SDK authentication flow" section for those two languages in the `auth.md` file. I also checked in the "live" docs just for giggles and the notes don't work there either for those languages. This note should be moved into the `auth.md` file for the languages that include this section, rather than in the main index file.

Reply:
Hrm, that doesn't really work either, as the placement of the notes then doesn't seem to make sense just for those languages. I'm going to delete it altogether.

Reply:
Ah scratch that, I'm going to add another stack selector to add in the link note, just for the languages where the section exists. Rather inelegant, but it works...